src/users.ts

```diff
@@ -51,8 +51,11 @@ export default class Users {
      * @returns {Object} - User object.
      * @example
      * const user = {
+     *  "email": "admin@example.com",
+     * "first_name": "John",
+     * "last_name": "Doe",
      * "credentials": {
-     *    "identity": "admin@example.com",
+     *    "username": "admin",
      *   "password": "12345678"
```

Comment:
SInce your interface credentials has username and secret shouldn't this be:
`"secret":  "12345678"` instead of `"password": "12345678"`
```
export interface credentials {
  username?: string
  secret?: string
}
```